pkg/build/kube/bazelbuildbits.go

```diff
@@ -56,74 +60,114 @@ func (b *BazelBuildBits) Build() error {
 	// make sure we cd back when done
 	defer os.Chdir(cwd)
 
+	// TODO(bentheelder): we assume the host arch, but cross compiling should
+	// be possible now
+	bazelGoosGoarch := fmt.Sprintf("%s_%s", runtime.GOOS, runtime.GOARCH)
+
 	// build artifacts
 	cmd := exec.Command(
 		"bazel", "build",
-		// TODO(bentheelder): we assume linux amd64, but we could select
-		// this based on Arch etc. throughout, this flag supports GOOS/GOARCH
-		"--platforms=@io_bazel_rules_go//go/toolchain:linux_amd64",
+		fmt.Sprintf("--platforms=@io_bazel_rules_go//go/toolchain:%s", bazelGoosGoarch),
 		// we want the debian packages
-		"//build/debs:debs",
+		//"//build/debs:debs",
+		// host installed binaries
```

Comment:
I'm confused by this comment - why are we building kubelet for the host?

Reply:
er yeah host is confusing here, what I mean is "binaries on the """"host""""" (node) as opposed to docker-wrapped binaries (the component images)

Reply:
re-worded the comment to reference the node